torch/distributed/optim/apply_optimizer_in_backward.py

```diff
@@ -88,3 +88,26 @@ def optimizer_hook(*_unused) -> None:
 
     for param in params:
         _apply_optimizer_in_backward_to_param(param)
+
+
+def _get_in_backward_optimizers(model: torch.nn.Module) -> List[torch.optim.Optimizer]:
```

Comment:
Just as a general comment: If we do not expect a function to be strictly called on the root module, I think a good convention is to make the arg name `module` (across all our APIs) instead of `model`. (For us, I think we mainly consider this in terms of composing with a sparse component or with other distributed APIs.)